posts/qrack_joins_uf.html

```diff
@@ -0,0 +1,169 @@
+                                <p>PyQrack has a commanding performance lead on this benchmark compared to other popular quantum computer simulators like Qiskit Aer (GPU) and QCGPU, where these three candidates are GPU-based simulators. Qiskit Aer (CPU) and Cirq (CPU) have been included on the chart, run on the same stock gaming laptop as the three GPU simulators, but remember that these software benchmarks are most fairly compared on identical hardware, whereas it isn’t possible for a CPU model and a GPU model to be the exact same hardware instance, obviously. However, PyQrack’s “hybrid CPU/GPU” methods utilize CPU simulation at the low qubit width end of the trend, which should be a fair comparison against CPU simulators until the user-selectable GPU switching threshold, which is 9 qubits in this case.</p>
+                                <p style="font-weight: bold"><strong>Qrack Architecture</strong></p>
+                                <p>Qrack is a collection of stacking and interchangeable optimization “layers,” which include OpenCL-based, scalable multi-device GPU, CPU, and GPU/CPU “hybrid” state vector simulation at base, “hybrid stabilizer” simulation with full and transparent recourse to state vector representation as fallback, an auxiliary “quantum binary decision tree” layer, and Schmidt decomposition techniques layered over stabilizer and state vector simulation for proactively and reactively identifying separable “cuts” in qubit subsystem representations to reduce memory footprint and improve execution speed. Floating point precision options available from source build include half, float, and double, with even limited support for quad math. By design, most users don’t need to worry about complicated custom configuration of the layers and settings at all; just instantiate a default C++11 “QInterface” or Python “QrackSimulator” class and go! The highly-optimized default layer stack will automatically leverage hybrid stabilizer simulation for a Clifford circuit or preamble, for example.</p>
```

Comment:
quotes around `layers` probably shouldn't encapsulate the comma, and I'm not sure they are needed.

Reply:
can remove the quotes